tools/build

```diff
@@ -2,7 +2,16 @@
 
 set -e
 
+verbose() [[ -v VERBOSE ]]
```

Comment:
This doesn't seem to work on my machine. I'd expect it to since I'm just on a stock Mac with stock Bash. Thoughts?

```console
$ tools/build
tools/build: line 5: conditional binary operator expected

$ bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
Copyright (C) 2007 Free Software Foundation, Inc.
```

Reply:
That's odd... Please try now, I rewrote it such that it should work on OSX bash too now

Reply:
Ok that works for me now, thanks!